virttest/utils_test/libvirt.py

```diff
@@ -922,6 +922,25 @@ def get_parts_list(session=None):
     return parts
 
 
+def get_disk_by_serial(session, serial_str):
+    """
+    Get disk by serial in VM
+
+    :param session: VM session 
+    :param serial_str: ID_SERIAL of disk, string value
+    :return: Disk name if find one with serial_str, else None
+    """
+    parts_list = get_parts_list(session)
+    for disk in parts_list:
+        cmd = "udevadm info --query=all --name=/dev/{0} | grep ID_SERIAL=".format(disk)
```

Comment:
Could you consider to use 'grep ID_SERIAL=serial_str'? That will make codes simpler.

Reply:
Updated.